core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -637,7 +637,10 @@ public void doTransactionRead(DB db)
 
 			fields=new HashSet<String>();
 			fields.add(fieldname);
-		}
+		} else if (dataintegrity) {
+      // pass the full field list if dataintegrity is on for verification
+      fields = new HashSet<String>(fieldnames);
```

Comment:
Yes. The issue is that line 640 (along with a mix of other lines) has tabs instead of spaces. In a configured IDE you cant tell, but looks like Github uses different spaces/tab. This should get fixed when the checkstyle change for core goes through. Didn't want to mix in any style changes.